server/webapp/WEB-INF/rails/webpack/views/components/switch/index.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as m from "mithril";
+import {MithrilComponent} from "jsx/mithril-component";
+import {Stream} from "mithril/stream";
+
+import {bind} from "classnames/bind";
+import * as styles from "./index.scss";
+
+const uuid4      = require("uuid/v4");
+const classnames = bind(styles);
+
+export interface Attrs {
```

Comment:
Can we use (or extend) the `LabelAttrs` interface as in `input_fields.tsx`? If it works for checkboxes, it should work for switches too.

Reply:
Sure.